scripts/check_lint.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+
+DOCKER_CMD=${DOCKER_CMD:-docker}
+
+if [ ! $(command -v ${DOCKER_CMD}) ]; then
+    exit 0
+fi
+
+echo "GolangCI Linter :: Started"
```

Comment:
check_license uses this format:
echo "Running check_license.sh"

(similarly for check_unit.sh)

Reply:
Updated https://github.com/hyperledger/aries-framework-go/compare/6a349ceee3704e5d4a1d8da0a776e7e4d5b83c5f..56569012d7114fec0ec68e653956803876587e8c